gapis/service/path/path.proto

```diff
@@ -491,6 +491,14 @@ message Thumbnail {
   bool disable_optimization = 7;
 }
 
+message OverrideConfig {
```

Comment:
It might be nice to make this more generic. Something more similar to having transforms be able to register them as "profiling overrides" with a name.

That way we don't have to change the public interface every time we add a what-if.